e2e_test/source/kafka.slt

```diff
@@ -139,6 +139,17 @@ create materialized source s10 with (
 statement ok
 select * from s10
 
+statement ok
+create materialized source s11 with (
+  connector = 'kafka', topic = 'proto_bin_c',
+  properties.bootstrap.server = '127.0.0.1:29092',
+  scan.startup.mode = 'earliest',
+  proto.message = 'test.User'
+) row format protobuf message 'test.User' row schema location 'file:///risingwave/proto-complex-schema.proto'
```

Comment:
please check if RW gets the right content. 
We do select here to start the executor and we check the results later in this file

Reply:
done